.codebuild.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
My only concern with this file is the name, who's to say codebuild doesn't eventually have a .codebuild.json default file. Maybe just name it something different?